### PR TITLE
AGW-2986: Tokenization Policy 1.1.0 Release Notes

### PR DESCRIPTION
**Pull request by LorenaSoledad**

### CODE REVIEW

**Comment by fermujica** [comment]
Thanks! 
I did a small change in the grammar of one of the sentences: b6575ff

The rest looks great! Already merged. 👍

File: modules/ROOT/pages/policies/detokenization-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+= Detokenization Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, detokenization, policy, release notes
```
Comment (evangelina) [inline modules/ROOT/pages/policies/detokenization-release-notes.adoc]: we can add tokenization here as well

File: modules/ROOT/pages/policies/tokenization-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+*Mule 4.1.1*
+
+=== New Features and Enhancements
+
+* Improved performance in header manipulation.
+
+=== Fixed Issues
+
+* Fixed error that caused the policy to apply the configuration in the response incorrectly.
```
Comment (evangelina) [inline modules/ROOT/pages/policies/tokenization-release-notes.adoc]: The fix message if confusing, shouldn't it be: Fixed error that caused to ignore the tokenization in the response?

File: modules/ROOT/pages/policies/detokenization-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+*Mule 4.1.1*
+
+=== New Features and Enhancements
+
+* Improved performance in header manipulation.
+
+=== Fixed Issues
+
+* Fixed error that caused the policy to apply the configuration in the response incorrectly.
```
Comment (evangelina) [inline modules/ROOT/pages/policies/detokenization-release-notes.adoc]: The fix message if confusing, shouldn't it be: Fixed error that caused to ignore the detokenization in the response?

**Pull request merged**